src/main/java/org/micks/champmaker/meals/MealEntity.java

```diff
@@ -0,0 +1,34 @@
+package org.micks.champmaker.meals;
+
+import javax.persistence.Entity;
+import javax.persistence.GeneratedValue;
+import javax.persistence.GenerationType;
+import javax.persistence.Id;
+import javax.persistence.Table;
+
+@Table(name = "meal")
+@Entity
+public class MealEntity {
+    @Id
+    @GeneratedValue(strategy = GenerationType.IDENTITY)
+    private Long id;
+
+    private String name;
```

Comment:
We should add also champId - it should be required for a meal.

Reply:
fixed